sycl/include/sycl/handler.hpp

```diff
@@ -318,45 +318,98 @@ checkValueRange(const T &V) {
 #endif
 }
 
-template <typename TransformedArgType, int Dims, typename KernelType>
-class RoundedRangeKernel {
+template <int Dims> class RoundedRangeIDGenerator {
+  id<Dims> Id;
+  range<Dims> OldRange;
+  range<Dims> NewRange;
```

Comment:
Namig variables "New"/"Old" does help readers to understand what values these variables to supposed to hold.
Do I understand it correctly that "OldRange" is the value uses passes to "parallel_for" call and "NewRange" is the rounded value DPC++ runtime passes to the unified runtime?
If so, I would prefer to name these something like "UserRange" and "RoundedRange".

Reply:
Correct, I think `UserRange` and `RoundedRange` are better descriptions